install-postgres.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+set -e
+set -u
+
+run_with_sudo() {
+    if [ $(id -u) -eq 0 ]; then
+        "$@"
+    else
+        sudo "$@"
+    fi
+}
+
+if dpkg -s postgresql &> /dev/null; then
```

Comment:
This looks interesting, keeps the script idempotent, would you suggest we add this elsewhere? 

However, I suppose adding a Dockerfile: 

- https://github.com/openupthecloud/open-cloud-dev-box/issues/29

Is the better longer term solution for these scripts.

Reply:
Great idea 🚀🚀🚀🚀🚀